alacritty_terminal/src/term/mod.rs

```diff
@@ -422,12 +423,14 @@ impl<'a, C> Iterator for RenderableCellsIter<'a, C> {
                     let mut cell = RenderableCell::new(self, cell);
                     cell.inner = RenderableCellContent::Cursor(self.cursor.key);
 
-                    // Only apply static color if it isn't close to the cell's current background.
-                    if let CellRgb::Rgb(color) = self.cursor.cursor_color {
-                        if color.contrast(cell.bg) >= MIN_CURSOR_CONTRAST {
-                            cell.fg = self.cursor.cursor_color.color(cell.fg, cell.bg);
-                        }
-                    }
+                    cell.fg = match self.cursor.cursor_color {
+                        // Only apply static color if it isn't close to the cell's current
+                        // background.
+                        CellRgb::Rgb(color) if color.contrast(cell.bg) < MIN_CURSOR_CONTRAST => {
+                            cell.fg
```

Comment:
Why would we set the cell.fg to cell.fg?

Reply:
I did it just for consistency with other `match` I've added, but there's no need to do it that way here, I guess.